src/models/OtrStore.js

```diff
@@ -0,0 +1,67 @@
+var _ = require("lodash");
+var DSA = require('otr').DSA;
+var OTR = require('otr').OTR;
+var Chan = require("./chan");
+
+module.exports = OtrStore;
+
+/** Stores OTR state for a given user
+ *
+ * Handels private key, current sessions and fingerprints.
+ */
+function OtrStore(name, client) {
+	_.merge(this, {
+		name: name,
+		privKey: undefined,
+		sessions: {},
+		client: client,
+	});
+	console.log('Generating DSA....');
+	this.privKey = new DSA();
+	console.log('OK, key generated');
+}
+
+/** Initialize an OTR session for query chans
+ *
+ * Event handler, to be called on chan creation
+ */
+OtrStore.prototype.initChan = function(args) {
+	var network_id = args.network;
+	var chan = args.chan;
+	if (chan.type == Chan.Type.QUERY) {
+		var network = _.find(this.client.networks, {id: network_id});
+		var session = this.getSession(chan.name, network);
+		var irc = network.irc;
+		if (session == undefined) {
+			session = new OTR({
+				fragment_size: 140,
+				send_interval: 200,
```

Comment:
> Could you maybe add a comment about these?

I did, actually (explaining why those values). BTW I'm not going to do all otr lib documentation here.

Being kicked of an IRC network for excessive flood is not science, so I see no better choice than experimentation.

(btw, no unit cause they were obvious imho : a time in network protocol is milliseconds, and a message size on a char-based protocol is in chars).